packages/rocketchat-channel-settings/client/views/channelSettings.coffee

```diff
@@ -106,4 +98,13 @@ Template.channelSettings.onCreated ->
 								return toastr.error TAPi18n.__(err.reason, err.details.roomType)
 							return toastr.error TAPi18n.__(err.reason)
 						toastr.success TAPi18n.__ 'Room_type_changed_successfully'
+			when 'archivationState'
+				if @$('input[name=archivationState]:checked').val() is 'true'
+					Meteor.call 'archiveRoom', @data?.rid, (err, results) ->
```

Comment:
What exactly do you mean by that @rodrigok? The first sentence contains a double negative, so I can't exactly make out what you're really saying...I assume it means something, but not 100% sure and we all know what they say about assuming things. :smile: